.travis.yml

```
@@ -86,6 +86,14 @@ script:
  - fold_unless_fail 'Run regression tests' 'regression' make test
  - fold_unless_fail 'Compile documentation and examples' 'compile_docs' make docs examples
  - fold_unless_fail 'Versions' 'versions' ./sile --debug=fonts,versions tests/bug-117.sil
+stages:
+  - test
+  - lint
+jobs:
+  include:
+    - stage: lint
+      name: "Lint commint messages"
+      script: fold_unless_fail 'Commitlint' 'commitlint' npx commitlint-travis
```

I think you need:

```suggestion
      script: fold_unless_fail 'Commitlint' 'commitlint' @commitlint/travis-cli
```

Personally, I would specify the version, so that builds don't start failing if the linter improves:

```suggestion
      script: fold_unless_fail 'Commitlint' 'commitlint' @commitlint/travis-cli@8.0.0
```

---

I appreciate the suggestion @rjmunro and I was honestly puzzled by how to execute the check, but your suggestion didn't work either.

---

Err, whoops. I didn't mean to drop the `npx` part.

```suggestion
      script: fold_unless_fail 'Commitlint' 'commitlint' npx @commitlint/travis-cli@8.0.0
```